vnext/local-cli/src/runWindows/utils/msbuildtools.ts

```diff
@@ -27,18 +27,18 @@ import {findLatestVsInstall} from './vsInstalls';
 export default class MSBuildTools {
   /**
    * @param version is something like 16.0 for 2019
-   * @param path  Path to MSBuild.exe (x86)
+   * @param msbuildPath  Path to MSBuild.exe (x86)
```

Comment:
>msbuildPath [](start = 12, length = 11)

OOI, what prompted the rename?

Reply:
nested properties called path.  Its a lint error.  Path from require(path)